docs/sources/clients/promtail/stages/replace.md

```diff
@@ -128,25 +128,25 @@ The log line would become
 
 ### With `replace` value in `template` format with hashing for obfuscating data
 
-To obfuscate sensitive data, you can combine the `replace` stage with the `Sha256` template method
+To obfuscate sensitive data, you can combine the `replace` stage with the `Sha2` or `Sha3` template method
```

Comment:
```suggestion
To hide sensitive data, you can combine the `replace` stage with the `Sha2` or `Sha3` template method.
```

Reply:
I think obfuscate better describes the intent of what we are doing with hashing data, hiding implies removing it or making it not visible anymore, obfuscate means we are intentionally making it difficult to know what the original text was.

Reply:
Will "obfuscate" be easily understood by non-native English speakers? I see that it is used specifically as code jargon.

When there is a choice between a simple word and a complicated word, we generally want to choose the simpler word.

Reply:
In this case obfuscate has an important meaning tied to its use in a security context; we are making the data more difficult to read but not impossible.

I think generally when working with things like PII and sensitive data the word obfuscate would be common, however it does appear to be more or less a synonym for "masking".

I don't really know how to comment on if a non native english speaker would be more likely to understand  "obfuscate" vs "mask" however I think either would be acceptable here.

Reply:
I generally prefer using common words over esoteric ones or jargon, but in this scenario I think _obfuscate_ is actually a better word because it insinuates that the data _could be understood_ if desired. Perhaps `mask` would satisfy this principle of least complexity and still retain the desired meaning?

This comment is a nitpick though -- any choice is fine :)

Reply:
It did pass my web search test, which is "see if I can figure it out easily by entering it in Google." I guess it is fine. 

I hope you will take my other suggestions, particularly adding the period to the end of the sentence. :)

Reply:
thx @oddlittlebird and @slim-bean. Implemented the other suggestions including the period. :)

Reply:
The discussion above proves that naming is, in fact, hard.

It also mirrors https://docs.google.com/document/d/1YN1nCWvfL4k6TnqwIqSEHmvMNSpmid3rXIP0YDXUFtc/edit in part.

"obfuscate" is well-defined in ITsec and we're NOT obfuscating it. We're offering the user to
* remove (replace with empty)
* replace
* hash
as of today. As those three words are well-defined, they are the only ones which make sense to use here.

Related: If users hash IPv4 addresses, anyone with a laptop can brute force the hash. That's the only possibly-stretched-into-fitting way to say "obfuscate" in the context of the whole thing: Well-bounded, well-known inputs with minuscule entropy.